ansible/deploy_riakcs_new.yml

```diff
@@ -0,0 +1,197 @@
+            or not (riak_conf_file.stat.exists | default(True))
+        else '' }}"
+    # HACK set fact because defaults cannot be loaded from other hosts
+    - set_fact: riak_nodename="{{ backend_vars.riak_nodename }}"
+
+- name: Configure swap for riak hosts
+  hosts: riakcs_new
+  roles:
+    - {role: swap}
+
+- name: Load admin keys from Riak CS conf
+  become: true
+  hosts: riakcs_new_control_host
+  vars_files:
+    - roles/riakcs/install/defaults/main.yml
+  tasks:
+    - include: roles/riakcs/get_admin_keys.yml tags=localsettings
+
+- name: Install Riak and Riak CS
+  become: true
+  hosts: riakcs_new
+  vars_files:
+    # for backend_vars.riak_version
+    - roles/riakcs/configure/vars/main.yml
+  roles:
+    - {role: riakcs/install, name: riak}
+    - {role: riakcs/install, name: riakcs, service: riak-cs}
+
+- name: Install Stanchion
+  become: true
+  hosts: stanchion_new
+  vars_files:
+    # for backend_vars.riak_version
+    - roles/riakcs/configure/vars/main.yml
+  roles:
+    - {role: riakcs/install, name: stanchion}
+
+- name: Configure and start Riak
+  # Riak CS must be installed before Riak can be started
+  become: true
+  hosts: riakcs_new
+  roles:
+    - role: riakcs/configure
+      name: riak
+      drop_conf: "{{ backend_vars.drop_conf }}"
+      advanced_conf: "{{ backend_vars.advanced_conf }}"
+      data_dirs:
+        - "{{ riak_data_dir }}"
+        - "{{ riak_ring_dir }}"
+        - "{{ riak_data_root_leveldb }}"
+        - "{{ riak_data_root_bitcask if _riak_backend == 'bitcask' else '' }}"
+        - "{{ riak_data_root_blocks if _riak_backend == 'leveldb' else '' }}"
+      when: _force_riak_config | bool
+
+- name: Start Riak
+  become: true
+  hosts: riakcs_new
+  tasks:
+    - service: name="riak" state=started enabled=yes
+      tags: after-reboot
+
+- name: Join Riak [CS] nodes to cluster
+  become: true
+  hosts: riakcs_new
+  tasks:
+    - include: roles/riakcs/join_cluster.yml
+      when: inventory_hostname != riakcs_control and _force_riak_config
+
+- name: Configure and start Stanchion
+  # Riak must be started before Stanchion can be started
+  become: true
+  hosts: stanchion_new
+  roles:
+    - role: riakcs/configure
+      name: stanchion
+      ulimit_n: 4096
+      data_dirs: [ "{{ stanchion_data_dir }}" ]
+
+- name: Configure and start Riak CS
+  become: true
+  hosts: riakcs_new
+  roles:
+    - {role: riakcs/configure, name: riakcs, service: riak-cs}
+
+- name: Create admin user
+  become: true
+  hosts: riakcs_new_control_host
+  roles:
+    - role: riakcs/create_admin_user
+      when: riak_key is undefined or riak_secret is undefined
+
+- name: Sync admin user keys to Riak CS cluster
+  become: true
+  hosts: riakcs_new
+  roles:
+    - {role: riakcs/sync_keys, name: riakcs}
+
+- name: Sync admin user keys to Stanchion
+  become: true
+  hosts:
+    - stanchion_new
+  roles:
+    - {role: riakcs/sync_keys, name: stanchion}
+
+- name: Remove Riak Restarter
```

Comment:
Is this still necessary?

Reply:
Probably can be removed since I don't think it's been applied everywhere. Leaving for a separate PR.